CHANGELOG

```diff
@@ -0,0 +1,4 @@
+2018/01/11	1.9.0 basdewitte	use bean property name for property key if omitted in annotation

```

Comment:
The version is incorrect. The latest versions which was released was `1.7`, which is why we're currently at `1.8-SNAPSHOT` in the `master`. :)

Could you please revert the change to the `pom.xml` file relating to this as well?